pkg/cloud/awscloud/awscloud.go

```diff
@@ -282,68 +247,65 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, rpmArch str
 	}
 
 	olog.Printf("[AWS] 🚚 Waiting for snapshot to finish importing: %s", *importTaskOutput.ImportTaskId)
-	err = WaitUntilImportSnapshotTaskCompleted(
-		a.ec2,
+	snapWaiter := newSnapshotImportedWaiterEC2(a.ec2)
+	snapWaitOutput, err := snapWaiter.WaitForOutput(
+		context.Background(),
 		&ec2.DescribeImportSnapshotTasksInput{
-			ImportTaskIds: []*string{
-				importTaskOutput.ImportTaskId,
+			ImportTaskIds: []string{
+				*importTaskOutput.ImportTaskId,
 			},
 		},
+		time.Hour*24,
```

Comment:
Non-configurable constant, just saying.

Reply:
Yes, this is consistent with the code in `osbuild-composer`. I can make this configurable if needed.

Reply:
Nah we are good, I could not find this constant in this patch so I thought it is being introduced as new. Thanks.